rust/gateway/src/main.rs

```diff
@@ -302,6 +320,10 @@ impl Cli {
     fn is_telemetry_allowed(&self) -> bool {
         !self.no_telemetry
     }
+
+    fn is_inc_buf_allowed(&self) -> bool {
+        !self.no_inc_buf
```

Comment:
I find it way easier to parse & think about flags when they're formed in *positive* way, like even as as an end user I'd rather configure `INCREASE_BUFFER=0` or `TELEMETRY=false` (btw: does `clap` parse `true` and `false` from these well? ). some goes for code, it's easier to read. 

then again, you're just conforming to existing convention here, so feel free to ignore this comment. :)